### DISCUSSION TITLE
Prisma function call does not return all fields

### DISCUSSION THREAD

**Discussion by XYZ321COD** [prisma/prisma#3315] [Q&A] [answered]
Hello, 
I'm trying to create new comment and connect it to already existing location and user in my application.  

This is how my type Comment looks like:
```javascript
type Comment {
  CommentID: ID!
  UserID: User!
  LocationID: Location!
  Context: String!
  Likes: [Like!]
  Medias: String
  createdAt: DateTime!
}
```
Creating new comment and connecting it to already existing user and location:
```javascript
      const userAuth = checkAuth(context);
      const newComment = await context.prisma.createComment({
        Context: args.comment.Context,
        Medias: args.comment.Medias,
        UserID: {
          connect: {
            UserID: userAuth.User.UserID,
          },
        },
        LocationID: {
          connect: {
            LocationID: "ckdu1ubl3000r0734u35r7f20",
          },
        },
      });
```
Am using debugger so I can check the value of ``` newComment  ```:
```javascript
       console.log(newComment);
```
So when am calling this mutation:
```javascript 
mutation createComment {
  createComment(comment: {
    Context: "Hello is the place open?",
    Medias : "SadPhoto.jpg",
  })
  {
    CommentID

  }
}
```

They value for ```newComment ``` is
```javascript
{CommentID: 'ckdx5u9gx007c0798o479f14p', Context: 'Hello is the place open?', Medias: 'SadPhoto.jpg', createdAt: '2020-08-16T14:04:06.128Z'}
```
So you can see that  User and Location values are missing. But when am call query provided by Prisma:
```javascript
{
  comment(where: {CommentID :"ckdx5u9gx007c0798o479f14p"}) {
    UserID {
      Login
    }
    LocationID{
      Name
    }
  }
}
```

I'm getting value for User and Location fields. Result:
```javascript
{
  "data": {
    "comment": {
      "UserID": {
        "Login": "User1"
      },
      "LocationID": {
        "Name": "Location1"
      }
    }
  }
}
```

How can I get access to value of User and Location fields from ```newComment ```?

**Comment by janpio** [comment]
What version of Prisma are you using?

**Reply by nikolasburk** [reply, marked as answer]
@XYZ321COD it seems like you're using Prisma 1 which is currently in [maintenance mode](https://github.com/prisma/prisma1/issues/4898), you can [raise this as an issue in the `prisma1` repo](https://github.com/prisma/prisma1).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
